pybotframework/botstate.py

```diff
@@ -0,0 +1,34 @@
+
+
+class BotStateMachine(object):
+
+    def __init__(self, states):
```

Comment:
What are the possible states if they are discrete?  Are we going off of https://github.com/google/bottery/ Actions?  Thanks.